pypowsybl/network/impl/network.py

```diff
@@ -3619,6 +3620,16 @@ def create_dangling_lines(self, df: DataFrame = None, **kwargs: ArrayLike) -> No
                 network.create_dangling_lines(id='BAT-1', voltage_level_id='VL1', bus_id='B1',
                                               p0=10, q0=3, r=0, x=5, g=0, b=1e-6)
         """
+        if df is not None:
```

Comment:
I need a review on that, I don't know if other ways to deprecate should be used.

Reply:
maybe you can try with a test

Reply:
I'll do it if you can't run test

Reply:
I tested in a jupyter notebook and it worked but I did not write any tests as I can't manage to make them run

Reply:
ok I test

Reply:
@geofjamg are we doing a breaking change or we maintain compatibility (can be much code for not so much)